README.md

```diff
@@ -71,6 +71,50 @@ gulpLoadPlugins({
 });
 ```
 
+## Multiple `config` locations
+
+While it's possile to grab plugins from another location, often times you may want to extend from another package that enables you to keep your own `package.json` free from duplicates, but still add in your own plugins that are needed for your project. Since the `config` option accepts an object, you can merge together multiple locations by using a utility function similar to the below:
+
+```js
+/**
+ * Recursively merge properties of two objects
+ * @param  {object} obj1
+ * @param  {object} obj2
+ * @return {object} Merged obj1 and obj2
+ */
+function mergeObjects(obj1, obj2) {
+  for (var p in obj2) {
+    try {
+      // Property in destination object set; update its value.
+      if (obj2[p].constructor == Object) {
+        obj1[p] = mergeObjects(obj1[p], obj2[p]);
+      } else {
+        obj1[p] = obj2[p];
+      }
+    } catch(e) {
+      // Property in destination object not set; create it and set its value.
+      obj1[p] = obj2[p];
+    }
+  }
+  return obj1;
+}
```

Comment:
If it's in documentation, I think it would be better to use a library like `lodash/merge` or `deepmerge` for clarity - this is a lot of non-gulp-load-plugins code.